Content.Server/VendingMachines/VendingMachineSystem.cs

```diff
@@ -384,7 +385,22 @@ private void EjectItem(EntityUid uid, VendingMachineComponent? vendComponent = n
                 return;
             }
 
-            var ent = Spawn(vendComponent.NextItemToEject, Transform(uid).Coordinates);
+            // Default spawn coordinates
+            var spawnCoordinates = Transform(uid).Coordinates;
```

Comment:
just cache the TransformComponent in a variable because you're getting it twice if the thing is wallmounted

Reply:
Changed the offset to use the var instead of calling it twice.